src/NServiceBus.Autofac.AcceptanceTests/NServiceBus.Autofac.AcceptanceTests.csproj

```diff
@@ -1,18 +1,21 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net452;netcoreapp2.0</TargetFrameworks>
-    <DebugSymbols>True</DebugSymbols>
-    <NoWarn>PCR0001</NoWarn>
+    <TargetFrameworks>net462;netcoreapp2.0</TargetFrameworks>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
```

Comment:
out of curiosity why `CopyLocalLockFileAssemblies`?

Reply:
> Add <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies> to all testing projects

From https://github.com/Particular/DotNetCoreLaunch/issues/33

Reply:
This puts all of the files into the bin folder and allows `dotnet test` to have what it needs to run the tests. `dotnet test` is how tests are run for .NET Core on Windows and Linux in the TC build process